### ISSUE TITLE
API resulst should contain a URL for the report

### ISSUE THREAD

**Issue by seccubus** [ssllabs/ssllabs-scan#61]
This allows tool programmers to redirect users to the correct SSL labs URL

**Comment by andrepereiradasilva** [comment]
I also agree with seccubus request.
For instance, having a test id and them we can construct the url from that id.

I don't know if this already exists.
Only know this URI parameters on https://www.ssllabs.com/ssltest/analyze.html:
- d: site domain
- hideResults: on (hide results from public view)
- clearCache: on (clean cache)
- latest: null (latest test?)
- s: ip of server (for multi server environment)

Is there other parameters in the URI query string, like, test id?

**Comment by ivanr** [comment]
I am happy to implement this, even though you can do this today. Just populate the "d" and "ignoreMismatch" parameters, optionally "hideResults".

@andrepereiradasilva, SSL Labs doesn't store test on long-term basis. Thus, we can't guarantee that the user will see the same results as in the scan. If the cache expires or a server is re-tested, the most recent results will show.

By the way, the "latest" parameter is used to retrieve a test that it still in progress. If this parameter is not used, the most recent complete test is shown.

**Comment by andrepereiradasilva** [comment]
@ivanr, oh i'm sorry to ear that, for research purposes, it would be good to preserve the test data.
So will have to store the json result instead.

**Comment by seccubus** [comment]
@ivanr to less this exercise is left to the API users, the less users are going to end up at the wrong endpoint. The URL can serve two purposes
a) Give the user a place to view the results
b) give the API a url where the HTML report can be downloaded

When I was scraping iso using the API I only had to change one url an in the report to actually make it portable.

**Comment by AshleyPoole** [comment]
I also agree with this request. It would be a great feature for people consuming the API.
